src/map/status_effect_container.cpp

```diff
@@ -151,20 +157,48 @@ namespace effects
 *                                                                       *
 ************************************************************************/
 
-CStatusEffectContainer::CStatusEffectContainer(CBattleEntity* PEntity)
+bool isSortedByStartTime(uint16 effectId) {
+    return effectId >= EFFECT_FIRE_MANEUVER && effectId <= EFFECT_DARK_MANEUVER;
+}
```

Comment:
Since I am not able to research/test/verify that any other effects are ordered like this, I will probably not be changing it in this PR (unless someone else can get confirmation on other status effects behaving like this).
If it is found out at a later date that this is how other certain status effects are ordered, the implementation can be changed at that point.

Thoughts?